src/clamav_report/clamav_report.py

```diff
@@ -287,6 +288,7 @@ def main() -> None:
         inventory_filename=validated_args["<inventory-file>"],
         hosts=validated_args["--group"],
         forks=validated_args["--forks"],
+        become=validated_args["--become"],
```

Comment:
Match the order of the `run_ansible()` input parameters:
```suggestion
        become=validated_args["--become"],
        hosts=validated_args["--group"],
        forks=validated_args["--forks"],
```